.travis.yml

```diff
@@ -106,19 +106,19 @@ before_install:
 install:
     - cd $girder_path
     - python setup.py develop
-    - pip install --upgrade 'git+https://github.com/cdeepakroy/ctk-cli'
-    - pip install -r $main_path/requirements.txt
+    - cd $large_image_path
+    - pip install -U -r requirements.txt
+    - python setup.py install
     - cd $main_path
-    - python setup.py build_ext --inplace
+    - pip install --upgrade 'git+https://github.com/cdeepakroy/ctk-cli'
+    - pip install -r requirements.txt
+    - python setup.py install
```

Comment:
If you do not generate and install a wheel, you have to install the package in development mode.

```
pip install -e .
```

or 

```
python setup.py develop
```